python-backend/app/api/mines/status/models/mine_status.py

```diff
@@ -55,18 +49,27 @@ def create_mine_status_labels_list(self):
                 self.mine_status_xref.mine_operation_status_sub_reason.description)
         return status_labels_list
 
-    def json(self, show_mgr=True):
-        status_values_list = self.create_mine_status_values_list()
-        status_labels_list = self.create_mine_status_labels_list()
-        return {
-            'mine_status_guid': str(self.mine_status_guid),
-            'mine_guid': str(self.mine_guid),
-            'mine_status_xref_guid': str(self.mine_status_xref.mine_status_xref_guid),
-            'status_values': status_values_list,
-            'status_labels': status_labels_list,
-            'effective_date': self.effective_date.isoformat(),
-            'expiry_date': self.expiry_date.isoformat()
-        }
+    def __repr__(self):
+        return '<MineStatus %r>' % self.mine_status_guid
+
+    def validate_status_code_exists(self, mine_status_xref, mine_status_code, code_or_description):
+        try:
+            return mine_status_xref[mine_status_code][code_or_description]
+        except KeyError:
+            return None
+
+    # def json(self, show_mgr=True):
```

Comment:
remove commented code?

Reply:
Un-commented as this function was used in other resources.